integration/__tests__/workspace.tests.ts

```diff
@@ -65,6 +66,19 @@ describe("Lens integration tests", () => {
       await app.client.waitForVisible(".ClustersMenu .ClusterIcon.active");
     });
 
+    it("shows workspace clusters in workspace overview", async () => {
+      await switchToWorkspace("default");
+      await app.client.click("[data-test-id=workspace-menu]");
+      await delay(500);
+      await app.client.click("[data-test-id=workspace-overview-menu-item]");
+      await app.client.waitUntilTextExists("h2", "default");
+      await app.client.waitUntilTextExists(".WorkspaceOverview .Table .TableRow .name", "minikube");
```

Comment:
Maybe we should specify less detailed css-selectors? e.g. `.WorkspaceOverview .name` or even better `[data-test-id=some_uniq_id]`?